doc/en/backwards-compatibility.rst

```diff
@@ -3,6 +3,57 @@
 Backwards Compatibility Policy
 ==============================
 
+.. versionadded: 6.0
+
+pytest is actively being evolved and as a project that is been decades in the making,
+we keep learning about new and better structures to express different details about testing.
+
+While we implement those we try to ensure easy transition and don't want to impose unnecessary churn on our users and community/plugin authors.
+
+We are considering multipe Types of backward compatibility considering transitions
+
+a) trivial: api's which trivially translate to the new mechanism,
+   and do not cause problematic details.
+
+   We try to support those indefinitely while encouraging users to switch to newer/better mechanisms.
```

Comment:
Here and several times below, I don't think these indents render e.g. https://github.com/pytest-dev/pytest/blob/de33fd5d357120588a295b59828da6918148fe0a/doc/en/backwards-compatibility.rst

```suggestion
We try to support those indefinitely while encouraging users to switch to newer/better mechanisms.
```

Reply:
just tried, those work fine

Reply:
Rechecking myself, I agree they work fine. Sorry for the false alarm!